### DISCUSSION TITLE
Field undifined

### DISCUSSION THREAD

**Discussion by hung292001** [pimcore/pimcore#17520] [Q&A / Support] [answered]
I have a class with one field "Name", so how i can get the data from that field to the web notify ?
![image](https://github.com/user-attachments/assets/fd27bc17-b183-4cda-939e-746c7327a183)
![image](https://github.com/user-attachments/assets/4bd8e91d-3b05-4b18-b0ab-9936ad627424)

**Comment by kingjia90** [comment, marked as answer]
`objectData` should be `e.detail.object.data.data.name`, `general` is mostly for system fields, suggesting also to `console.log(e.detail.object.data)` and check the object structure.

Hope it helps

**Reply by hung292001** [reply]
Thanks for your help; I made it.
